docs/examples/machine_translation/transformer.md

```diff
@@ -241,7 +241,7 @@ print(sample_tgt_seq)
 ```
 
 If you'd like to train your own transformer models, you may find the training scripts in our
-[model zoo](../../model_zoo/machine_translation/index.rst).
+[model zoo](../../../scripts/machine_translation/index.rst).
```

Comment:
Without this change we run into https://github.com/rtfd/recommonmark/issues/93

Reply:
But it is not valid for the CI build with publication `/var/lib/jenkins/workspace/gluon-nlp-docs/docs/examples/machine_translation/transformer.ipynb.rst:link outside of source directory: '../scripts/machine_translation/index.rst'`